tests/models/resnet/test_modeling_resnet.py

```diff
@@ -110,6 +113,20 @@ def create_and_check_for_image_classification(self, config, pixel_values, labels
         result = model(pixel_values, labels=labels)
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.num_labels))
 
+    def create_and_check_backbone(self, config, pixel_values, labels):
+        model = ResNetBackbone(config=config)
+        model.to(torch_device)
+        model.eval()
+        result = model(pixel_values)
+
+        # verify hidden states
+        self.parent.assertEqual(len(result.hidden_states), 4)
+        self.parent.assertListEqual(result.stage_names, config.out_features)
+        self.parent.assertListEqual(list(result.hidden_states[0].shape), [3, 10, 8, 8])
+
+        # verify channels
+        self.parent.assertListEqual(model.channels, config.hidden_sizes)
```

Comment:
Can we also check the strides?

Reply:
Actually both DETR and MaskFormer don't require the stride information, can I remove the `strides` property for now, and only keep the bare minimum?

Reply:
Yes, no need to have it if it's not useful for now.